geode-core/src/main/java/org/apache/geode/management/internal/cli/CliUtil.java

```diff
@@ -212,8 +211,7 @@ public static void bytesToFiles(byte[][] fileData, String parentDirPath, boolean
   }
 
   public static boolean isValidFileName(String filePath, String extension) {
-    boolean isValid = true;
-    return isValid;
+    return true;
```

Comment:
Can we just delete this method entirely?

Reply:
Looks unused.  Must've just condensed the return from IntelliJ's inspection window and not looked too closely at this silly method.